### DISCUSSION TITLE
云原生的弹性 AI 训练系列之三：借助弹性伸缩的 JUPYTER NOTEBOOK，大幅提高 GPU 利用率 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#22] [General]
# 云原生的弹性 AI 训练系列之三：借助弹性伸缩的 JUPYTER NOTEBOOK，大幅提高 GPU 利用率 | 高策

江湖小虾米

https://gaocegege.com/Blog/kubernetes/elastic-jupyter

**Comment by Pan-will** [comment]
作者您好！我尝试通过kubectl apply 创建JupyterNotebook这个CR，但是会报如下Error：
`error: unable to recognize "notebook-cr.yaml": no matches for kind "JupyterNotebook" in version "kubeflow.tkestack.io/v1alpha1"`，请问对应的JupyterNotebook和JupyterGateway 这两个kind怎么定义呢？

**Reply by Pan-will** [reply]
我在`https://github.com/skai-x/elastic-jupyter-operator`中未能找到JupyterNotebook和JupyterGateway 这两个CRD的YAML文件，或者您能提供一个获取方式吗，万分感谢！

**Reply by gaocegege** [reply]
项目已经不再维护了，我不是非常建议你使用它

**Reply by Pan-will** [reply]
感谢策哥！还奇怪github上Issue没有讨论来着~
这个项目的思路几乎完美解决我现在遇到的GPU利用率低的问题，请问是有什么漏洞放弃维护了吗

**Reply by gaocegege** [reply]
Hi 与技术无关，项目是在腾讯时设计的，目前我已经离开腾讯
